packages/react-data-grid-addons/src/draggable-header/__tests__/DraggableContainer.spec.js

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import { shallow } from 'enzyme';
 
 import DraggableContainer from '../DraggableContainer';
-import ReactDataGrid from 'react-data-grid';
+import DataGrid from 'react-data-grid';
 
 describe('<DraggableContainer />', () => {
   it('should render grid wrapper to be used as Drag and Drop context with grid as a child component inside', () => {
     const wrapper = shallow(<DraggableContainer />);
-    expect(wrapper.find(ReactDataGrid));
+    expect(wrapper.find(DataGrid));
```

Comment:
Don't know what this test is for....

Reply:
It doesn't even have a `toBeDefined()` or anything, so I've removed this test file and another useless one.
The tests are in a dire state right now, and we'll need to overhaul them, but not in this PR.

Reply:
Agree. This reminds me a jest eslint rule that to always require assertion.